wdn/templates_5.0/js/main.js

```diff
@@ -7,3 +7,51 @@ requirejs.config({
 		}
 	}
 });
+
+require(['jquery'], function($) {
```

Comment:
This should probably live in a file named something like navigation.js instead of main.js.